templates/v2-magazine/v2-magazine.js

```diff
@@ -50,9 +50,9 @@ const buildMetaAuthor = () => {
 
 // Build the 2nd element of the metadata row
 const buildMetaPubDate = () => {
-  const { DATE_LANGUAGE, DATE_OPTIONS, DATE_OPTIONS_MOBILE } = MAGAZINE_CONFIGS;
+  const { DATE_OPTIONS, DATE_OPTIONS_MOBILE } = MAGAZINE_CONFIGS;
   let pubDate = getMetadata('publish-date');
-  const locale = getMetadata('locale') || DATE_LANGUAGE;
+  const locale = getMetadata('locale');
```

Comment:
We need to talk about this separately after.

Reply:
ok sure, but the variable DATE_LANGUAGE is no more there right? or we keep it as a backup?